src/pages/Contribute.js

```diff
@@ -24,49 +69,64 @@ export default function Contribute() {
   }
 
   return (
-    <form onSubmit={handeleSubmit}>
-      <h1>Which article inspires you?</h1>
-      <h3>Category?</h3>
-      <input
-        autoFocus
-        type="text"
-        value={category}
-        onChange={event => setCategory(event.target.value)}
-      />
-      <h3>Headline</h3>
-      <input
-        autoFocus
-        type="text"
-        value={headline}
-        onChange={event => setHeadline(event.target.value)}
-      />
-      <h3>Paragraph</h3>
-      <input
-        autoFocus
-        type="text"
-        value={paragraph}
-        onChange={event => setParagraph(event.target.value)}
-      />
-      <h3>Rating</h3>
-      <div>
-        {[1, 2, 3, 4, 5].map(value => (
+    <ContributionContainer>
+      <ContributionForm onSubmit={handleSubmit}>
+        <ContributionHeadline>Which article inspires you?</ContributionHeadline>
+        <InformationLabel>
+          Headline
+          <input
+            placeholder="e.g. Worldpeace has been acquired"
+            type="text"
+            value={headline}
+            onChange={event => setHeadline(event.target.value)}
+          />
+        </InformationLabel>
+        <InformationLabel>
+          Category
+          <select
+            value={category}
+            onChange={event => setCategory(event.target.value)}
+          >
+            <option value="">choose category</option>
+            <option value="economy">Economy</option>
+            <option value="tech">Tech</option>
+            <option value="sport">Sports</option>
+            <option value="lifestyle">Lifestyle</option>
+          </select>
+        </InformationLabel>
+        <InformationLabel>
+          Paragraph
+          <textarea
+            placeholder="Please provide the first paragraph of the article here"
+            type="text"
+            rows="5"
+            value={paragraph}
+            onChange={event => setParagraph(event.target.value)}
+          />
+        </InformationLabel>
+        <InformationLabel>
+          Rating
+          {[1, 2, 3, 4, 5].map(value => (
+            <input
+              key={value}
+              type="button"
+              value={value}
+              active={value === rating}
+              onClick={() => setRating(value)}
+            />
+          ))}
+        </InformationLabel>
```

Comment:
Fehlermeldung: 

Warning: Received `false` for a non-boolean attribute `active`.

If you want to write it to the DOM, pass a string instead: active="false" or active={value.toString()}.

If you used to conditionally omit it with active={condition && value}, pass active={condition ? value : undefined} instead.
    in input (at Contribute.js:110)
    in label (created by Context.Consumer)
    in Styled(label) (at Contribute.js:107)
    in form (created by Context.Consumer)
    in Styled(form) (at Contribute.js:73)
    in div (created by Context.Consumer)
    in Styled(div) (at Contribute.js:72)
    in Contribute (created by Context.Consumer)
    in Route (at App.js:40)
    in Switch (at App.js:38)
    in main (created by Context.Consumer)
    in Styled(main) (at App.js:37)
    in Router (created by BrowserRouter)
    in BrowserRouter (at App.js:35)
    in div (created by Context.Consumer)
    in Styled(div) (at App.js:33)
    in App (at src/index.js:6)